kediatr-core/pom.xml

```diff
@@ -223,6 +223,13 @@
             <artifactId>kotlinx-coroutines-core</artifactId>
             <version>1.3.1</version>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-reflect</artifactId>
+            <version>1.5.21</version>
```

Comment:
If you wish, you can use the kotlin-bom to allign all Kotlin SDK libraries with the same version.